### DISCUSSION TITLE
academy/puppeteer-playwright/common-use-cases/logging-into-a-website

### DISCUSSION THREAD

**Discussion by giscus** [apify/apify-docs#693] [Comments]
# academy/puppeteer-playwright/common-use-cases/logging-into-a-website

Learn about some of the most common uses cases of Playwright and Puppeteer, and how to handle these use cases when you run into them.

https://docs.apify.com/academy/puppeteer-playwright/common-use-cases/logging-into-a-website

**Comment by alekseihyatt** [comment]
Please make guide with Token Authentication
